### PR TITLE
Docs for conditionals

### PR DESCRIPTION
**Pull request by plaindocs**
Slightly improved docs. 

Skipping Repository Settings Screenshots because we're going to phase those out or reduce them in #1039

### CODE REVIEW

File: pages/pipelines/conditionals.md.erb [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
-The `if` parameter provides the ability to run steps only when specific conditions are met. Using C-like expressions, boolean conditions are defined to restrict the running of steps to specific circumstances. Note that we do not support Build Meta-data for conditionals.
+Using conditionals, you can run builds or steps only when specific conditions are met. Define boolean conditions using C-like expressions. Note that you cannot use [Build Meta-data](/docs/pipelines/build-meta-data) in conditional expressions.
+
+You can define conditionals at the step level in your `pipeline.yml` or at the pipeline level in your Buildkite version control provider settings.
 
 {:toc}
 
+## Conditionals in Pipelines
+
+Use conditionals in your pipeline settings (_Repository Providers_) to run builds run builds only when expressions evaluates to `true`. For example, to run only when a branch is tagged `Approved`:
```
Comment (sj26) [inline pages/pipelines/conditionals.md.erb:11]: What's the "Repository Providers" here about?

Comment (sj26) [reply pages/pipelines/conditionals.md.erb:11]:
```suggestion
Use conditionals in your pipeline settings (_Repository Providers_) to run builds only when expressions evaluates to `true`. For example, to run only when a branch is tagged `Approved`:
```

Comment (plaindocs) [reply pages/pipelines/conditionals.md.erb]: Thanks for the :eyes: @sj26 - yup I mixed the Repository settings, will fix.

**Comment by harrietgrace** [comment]
I've shuffled the scm provider conditionals bit of the REST API pipelines page into its own little section. It was previously in the 'github only' section, but now it works for all providers 👯

I also removed the line about other providers having no settings, because now they do! 🎉

**Comment by harrietgrace** [comment]
@plaindocs if you're happy with this, feel free to merge 🌈

**Review by harrietgrace — APPROVED** [review]
✨

**Comment by plaindocs** [comment]
Thanks @harrietgrace !

**Pull request merged**